protocol/lavaprotocol/request_builder.go

```diff
@@ -131,93 +125,31 @@ func ReplaceRequestedBlock(requestedBlock int64, latestBlock int64) int64 {
 	}
 	relayRequest := &pairingtypes.RelayRequest{
-		RelayData:       relayRequestData,
-		RelaySession:    dataReliabilityRelaySession(lavaChainID, relayRequestData, chainID, providerPublicAddress, epoch, relayNum),
-		DataReliability: vrfData,
+		RelayData:    relayRequestData,
+		RelaySession: dataReliabilityRelaySession(lavaChainID, relayRequestData, chainID, providerPublicAddress, epoch, relayNum),
 	}
 	sig, err := sigs.SignRelay(privKey, *relayRequest.RelaySession)
 	if err != nil {
 		return nil, err
 	}
 	relayRequest.RelaySession.Sig = sig
 
-	sig, err = sigs.SignVRFData(privKey, relayRequest.DataReliability)
-	if err != nil {
-		return nil, err
-	}
-	relayRequest.DataReliability.Sig = sig
 	return relayRequest, nil
 }
 
-func VerifyReliabilityResults(originalResult *RelayResult, dataReliabilityResults []*RelayResult, totalNumberOfSessions int) (conflict bool, conflicts []*conflicttypes.ResponseConflict) {
-	verificationsLength := len(dataReliabilityResults)
-	participatingProviders := make([]utils.Attribute, verificationsLength+1) // only used for logging
-	participatingProviders = append(participatingProviders, utils.Attribute{Key: "originalAddress", Value: originalResult.ProviderAddress})
-	for idx, dataReliabilityResult := range dataReliabilityResults {
-		add := dataReliabilityResult.ProviderAddress
-		participatingProviders = append(participatingProviders, utils.Attribute{Key: "address" + strconv.Itoa(idx), Value: add})
-		conflict_now, detectionMessage := compareRelaysFindConflict(originalResult, dataReliabilityResult)
-		if conflict_now {
-			conflicts = []*conflicttypes.ResponseConflict{detectionMessage}
-			conflict = true
-		}
-	}
-	if conflict {
-		// CompareRelaysAndReportConflict to each one of the data reliability relays to confirm that the first relay was'nt ok
-		for idx1 := 0; idx1 < verificationsLength; idx1++ {
-			for idx2 := (idx1 + 1); idx2 < verificationsLength; idx2++ {
-				conflict_responses, moreDetectionMessages := compareRelaysFindConflict(dataReliabilityResults[idx1], dataReliabilityResults[idx2])
-				if conflict_responses {
-					conflicts = append(conflicts, moreDetectionMessages)
-				}
-			}
-		}
-	}
-
-	if !conflict && totalNumberOfSessions == verificationsLength { // if no conflict was detected data reliability was successful
-		// all reliability sessions succeeded
-		utils.LavaFormatInfo("Reliability verified successfully!", participatingProviders...)
-	} else {
-		utils.LavaFormatInfo("Data is not Reliability verified!", participatingProviders...)
+func VerifyReliabilityResults(originalResult *RelayResult, dataReliabilityResult *RelayResult) (conflicts *conflicttypes.ResponseConflict) {
+	conflict_now, detectionMessage := compareRelaysFindConflict(originalResult, dataReliabilityResult)
+	if conflict_now {
+		utils.LavaFormatInfo("Reliability verified successfully!")
```

Comment:
this is wrong, if there is a conflict it's a warning saying there is discrepancy, and if there isn't it should say verified successfully